site/src/pages/CreateWorkspacePage/CreateWorkspacePage.tsx

```diff
@@ -1,27 +1,39 @@
 import { useMachine } from "@xstate/react"
```

Comment:
I did some basic testing of the feature, and it looks like "Open in Coder" link contains parameters that the user has clicked in the form.

Shouldn't it contain all values even if they are default?

Reply:
I don't think so. If the user does not provide a value, I would expect we automatically get the default if it is provided. cc.: @bpmct wdyt?

Reply:
What if default values change over time :) Do we want to stick to the one present at the link generation or use the fresh one?

Reply:
If not specified, we should use the default of the template at the time the button was clicked, not generated. So if the default changes over time, different users will see different behaviors, but that is OK.